packages/venia-concept/src/components/ProductQuantity/quantity.css

```diff
@@ -2,3 +2,41 @@
     display: flex;
     flex-direction: column;
 }
+.quantityFieldbox button {
+    margin: 0 2px;
```

Comment:
The spacing could use some work. Once you switch the icons over to the `Icon` component, can you try to make them centered appropriately? You may need to experiment with flexbox here.

[![Image from Gyazo](https://i.gyazo.com/758e5a0830366079887c24307459ad60.png)](https://gyazo.com/758e5a0830366079887c24307459ad60)